burn-ndarray/src/ops/int_tensor.rs

```diff
@@ -377,4 +378,20 @@ impl<E: FloatNdArrayElement> IntTensorOps<Self> for NdArray<E> {
     ) -> <NdArray<E> as Backend>::IntTensorPrimitive<D> {
         NdArrayOps::swap_dims(tensor, dim1, dim2)
     }
+
+    fn int_random<const D: usize>(
+        shape: Shape<D>,
+        distribution: Distribution,
+        device: &NdArrayDevice,
+    ) -> NdArrayTensor<i64, D> {
+        let mut seed = SEED.lock().unwrap();
+        let mut rng = if let Some(rng_seeded) = seed.as_ref() {
+            rng_seeded.clone()
+        } else {
+            get_seeded_rng()
+        };
+        let tensor = Self::int_from_data(Data::random(shape, distribution, &mut rng), device);
```

Comment:
Not sure what default distribution will do here? I suppose it will give floats in [0,1[ then floor them all to zero?